lib/administrate/field/deferred.rb

```diff
@@ -11,7 +11,8 @@ def initialize(deferred_class, options = {})
       attr_reader :deferred_class, :options
 
       def new(*args)
-        deferred_class.new(*args, options)
+        new_options = if args.last.is_a? Hash then args.pop else {} end
```

Comment:
Heh - I actually usually use ternaries but I recently discovered the "verbose ternary with `then`" and I like it a lot more. I was happy to switch though.

However, moving to respond_to? caused tests to fail, presumably because other objects that are not hashes have a `pop` method. It really needs to be a Hash here because the whole point of this is to trap Ruby's Hash-as-final-argument syntax which was used for positional params prior to 2.2. 

I could *probably* write `args.respond_to?(:pop) && args.respond_to?(:merge)` but it seems clumsy and actually not saying what we're trying to say here.